docker/Dockerfile

```diff
@@ -97,7 +97,7 @@ COPY deployments/gpu-operator/charts/node-feature-discovery/crds/nfd-api-crds.ya
 # Install / upgrade packages here that are required to resolve CVEs
 ARG CVE_UPDATES
```

Comment:
In general, I would recommend removing this block entirely. We have largely stopped using this envvar across our projects -- especially since we've streamlined our waiver process.

Reply:
Yes, agreed . I think this block was added as a stopgap to quickly fix CVEs at the time

Reply:
I can make this change in a follow up PR